frontend/package.json

```diff
@@ -4,28 +4,29 @@
   "private": true,
   "scripts": {
     "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
+    "build": "npm run license-file-headers-check && vue-cli-service build",
```

Comment:
as this is now a build step this should not be needed anymore

Reply:
Removed in most recent commit